part2/style/style.css

```diff
@@ -0,0 +1,164 @@
+      format("woff"),
+    url("https://nomail.com.ua/files/woff2/77c39d7d46ba8c98529571e14b59cb99.woff2")
+      format("woff2");
+}
+@font-face {
+  font-family: Georgia italic;
+  src: url("https://nomail.com.ua/files/eot/35ac14f8a045f0a3f8f171d6b32dd09f.eot?#iefix")
+      format("embedded-opentype"),
+    url("https://nomail.com.ua/files/woff/35ac14f8a045f0a3f8f171d6b32dd09f.woff")
+      format("woff"),
+    url("https://nomail.com.ua/files/woff2/35ac14f8a045f0a3f8f171d6b32dd09f.woff2")
+      format("woff2");
+}
+@font-face {
+  font-family: "Georgia";
+  src: url("https://nomail.com.ua/files/eot/de6243382a94fdc4813e4cbd4ff7d919.eot?#iefix")
+      format("embedded-opentype"),
+    url("https://nomail.com.ua/files/woff/de6243382a94fdc4813e4cbd4ff7d919.woff")
+      format("woff"),
+    url("https://nomail.com.ua/files/woff2/de6243382a94fdc4813e4cbd4ff7d919.woff2")
+      format("woff2");
+}
+.html {
+  background-color: #ffffff;
+}
+.page {
+  width: 1200px;
+  height: 100vh;
+  background-color: #464f54;
+  margin: 0 auto;
+  position: relative;
+}
+.top-section {
+  width: 876px;
+  height: 187px;
+  box-shadow: -4px -4px 59px rgba(0, 0, 0, 0.16);
+  background-color: #181818;
+  position: absolute;
+  top: 47px;
+  right: 0;
+  z-index: 1;
+  padding-left: 64px;
+}
+.top-section__title {
+  color: #767676;
+  font-family: "Proxima Nova";
+  font-size: 15px;
+  text-transform: uppercase;
+  letter-spacing: 4.8px;
+  font-weight: 400;
+  margin-top: 50px;
+}
+.top-section__input {
+  color: #fefefe;
+  font-size: 48px;
+  font-family: "Playfair Display";
+  font-weight: 400;
+  margin: 0;
+  border-right: solid 2px #edeff2;
+  max-width: 350px;
+}
+.top-section__line {
+  color: #ffffff;
+  opacity: 0.1;
+}
+.selector {
+  z-index: 1;
+  width: 812px;
+  height: 76px;
+  box-shadow: -4px -4px 59px rgba(0, 0, 0, 0.16);
+  background-color: #fbc02d;
+  position: absolute;
+  top: 234px;
+  right: 0;
+}
+.selector__ul-wrapper {
+  position: absolute;
+  left: 42px;
+  fill: #ffffff;
+}
+.selector__item {
+  display: inline-block;
+  list-style-type: none;
+  margin-right: 22px;
+  cursor: pointer;
+  transition: 500ms;
+}
+.selector__ul-wrapper:hover .selector__item:not(:hover) {
+  fill-opacity: 0.5;
+}
+.main-section {
+  width: 1060px;
+  height: 751px;
+  box-shadow: 0 0 35px rgba(0, 0, 0, 0.22);
+  background-color: #ffffff;
+  position: absolute;
+  bottom: 0;
+  left: calc(50% - 530px);
+  display: flex;
+}
+.main-section__left-border {
+  flex-basis: 67px;
+  height: 751px;
+  border-right: 2px solid #e8eced;
+  margin-right: 56px;
+}
+.main-section__text-field {
+  position: absolute;
+  top: 190px;
+  left: 125px;
+}
+.text-field__author {
+  color: #b4b4b4;
+  font-family: "Proxima Nova";
+  font-size: 20px;
+  text-transform: uppercase;
+  letter-spacing: 6.72px;
+  font-weight: lighter;
+}
+.text-field__title {
+  font-family: "Futura PT - Heavy";
```

Comment:
It is nice and better to have your font fall back, just in case the main font can't be loaded. So try to add at least one similar to 'Futura PT'. Maybe use font a bit similar to this font or just leave `font-family: "Futura PT - Heavy", sans-serif;` if not sure.